WordPress/WordPressUITests/Tests/ReaderTests.swift

```diff
@@ -0,0 +1,36 @@
+import UITestsFoundation
+import XCTest
+
+class ReaderTests: XCTestCase {
+    private var readerScreen: ReaderScreen!
+
+    override func setUpWithError() throws {
+        setUpTestSuite()
+
+        _ = try LoginFlow.loginIfNeeded(siteUrl: WPUITestCredentials.testWPcomSiteAddress, email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
+        readerScreen = try EditorFlow
+            .goToMySiteScreen()
+            .tabBar.goToReaderScreen()
+    }
+
+    override func tearDownWithError() throws {
+        takeScreenshotOfFailedTest()
+        if readerScreen != nil && !TabNavComponent.isVisible() {
```

Comment:
> If this is so, what do you think of renaming the method to reflect this?
The idea was to have it more generic but you are right about the `.fast`. It wouldn't make much difference in this case though, because the posts are mocked and we have only 3 of them.